schema/transportation/segment.yaml

```diff
@@ -62,6 +63,78 @@ properties:
       routes: { "$ref": "#/$defs/propertyDefinitions/routes" }
 "$defs":
   propertyDefinitions:
+    destinationLabelType:
+      description: >-
+        The type of object of the destination label.
+      type: string
+      enum: 
+      - street
```

Comment:
should we have `place` as well? it's at least part of your example: https://github.com/OvertureMaps/schema/pull/176/files#diff-83cf2a8a673bc7660b22d82ec08a79d2c5412a5f700b4315ca51ba9828fb2d44

Reply:
fixed